{{ cookiecutter.project_slug }}/README.md

```diff
@@ -11,6 +11,13 @@ Currently supporting only Python 3.x.
 **IMPORTANT**:
 Docker Compose is used _just_ for development environment. The Dockerfile works without it.
 
```

Comment:
maybe we can have a README outside the folder `{{cookiecutter.project_slug}}` with instructions to clone and use the boilerplate and in this file all about the future project (like a model), is it possible?